flocker/acceptance/testtools.py

```diff
@@ -858,3 +859,34 @@ def clean(cluster):
             return calling_test_method
         return wrapper
     return decorator
+
+
+def create_dataset(test_case, cluster,
+                   maximum_size=REALISTIC_BLOCKDEVICE_SIZE):
```

Comment:
Might be useful, yeah, but for now this is just refactoring existing tests. When someone needs that they can add it.